composer.json

```diff
@@ -14,14 +14,14 @@
     },
     "require": {
         "react/react": "~0.4",
-        "symfony/console": "~2.6",
-        "symfony/expression-language": "~2.6",
-        "symfony/debug": "~2.6"
+        "symfony/console": "2.6 - 3.0",
```

Comment:
That would be wrong either, because that would mean 2.6.0-Infinity and we don't want to use symfony 4.0 automatically when it comes out as it can have breaking changes. This is how [semver](http://semver.org/) works.